### PR TITLE
PLATY-502: Use MdcLoggingExecutionContext to propagate "file referenc…

### PR DESCRIPTION
**Pull request by cybersaurus**
…e" value between execution contexts and include in logging statements via MDC.

### CODE REVIEW

File: app/services/FileNotificationDetailsRetriever.scala [hunk 1]
```diff
@@ -21,6 +21,6 @@ import model.{QuarantinedFile, S3ObjectLocation, UploadedFile}
 import scala.concurrent.Future
 
 trait FileNotificationDetailsRetriever {
-  def retrieveUploadedFileDetails(objectLocation: S3ObjectLocation): Future[UploadedFile]
-  def retrieveQuarantinedFileDetails(objectLocation: S3ObjectLocation): Future[QuarantinedFile]
+  def retrieveUploadedFileDetails(implicit objectLocation: S3ObjectLocation): Future[UploadedFile]
```
Comment (tomasz-rosiek) [inline app/services/FileNotificationDetailsRetriever.scala]: Why implicit here?

File: app/services/ContinuousPoller.scala [hunk 2]
```diff
@@ -83,7 +89,7 @@ class ContinuousPollingActor(job: PollingJob, retryInterval: FiniteDuration) ext
 
 object ContinuousPollingActor {
 
-  def apply(orchestrator: PollingJob, retryInterval: FiniteDuration): Props =
+  def apply(orchestrator: PollingJob, retryInterval: FiniteDuration)(implicit executionContext: ExecutionContext): Props =
```
Comment (tomasz-rosiek) [inline app/services/ContinuousPoller.scala]: And here?

File: app/services/FileManager.scala [hunk 3]
```diff
@@ -26,8 +26,8 @@ case class ObjectWithMetadata(content: String, metadata: ObjectMetadata)
 
 trait FileManager {
 
-  def retrieveMetadata(objectLocation: S3ObjectLocation): Future[ObjectMetadata]
+  def retrieveMetadata(implicit objectLocation: S3ObjectLocation): Future[ObjectMetadata]
```
Comment (tomasz-rosiek) [inline app/services/FileManager.scala]: Why implicit here?

File: app/services/FileNotificationDetailsRetriever.scala (hunk 1 above)
Comment (cybersaurus) [reply app/services/FileNotificationDetailsRetriever.scala]: The implementation of that method needs an implicit S3ObjectLocation in order to call an implicit function - util.logging.LoggingDetails.fromS3ObjectLocation.

**Pull request merged**